src/error.rs

```diff
@@ -21,9 +67,46 @@ macro_rules! implement {
             fn erfc(self) -> Self {
                 unsafe { math::erfc(self as f64) as Self }
             }
+
+            #[inline]
+            fn erfinv(self) -> Self {
+                erfinv(self as f64) as Self
```

Comment:
My main concern is that this conversion. There should be two functions generated: one for `f32` and one for `f64`. Have a look at `beta.rs`, for instance.